duo-client/src/main/java/com/duosecurity/client/Http.java

```diff
@@ -326,6 +334,23 @@ private String canonQueryString()
     return Util.join(args.toArray(), "&");
   }
 
+  private String canonJSONBody(){
+    JSONObject jsonBody = new JSONObject(params);
+    return jsonBody.toString();
```

Comment:
The `requestBoday.create` take straight JSONObject.toString() as json body content, but Duo's json canon is slightly different as the below method `canonJSONString()`. Duo's canon json is more pythonic output. Java JSONObject contain `{}` in the outer layer.